index.html

```diff
@@ -112,6 +112,14 @@ <h2>
       <li>Requesting a [=digital credential=], including mechanisms for secure
       presentation.
       </li>
+      <li>Ensuring that when an API call is made, the website does not learn
+      anything about the a holder's [=digital credentials=] or their software
+      unless the [=user agent=] has gained explicit user consent.
```

Comment:
Maybe better:

```suggestion
      unless the [=user agent=] has express user consent.
```

aligns a bit more with permissions API's [express permission](https://www.w3.org/TR/permissions/#dfn-express-permission)